memo_app.rb

```diff
@@ -2,9 +2,48 @@
 
 require 'sinatra'
 require 'json'
+require 'pg'
+require 'dotenv'
+
+Dotenv.load
+
+# MemoClass for Run SQL
+class Memo
+  class << self
+    def connection
+      PG.connect(
+        host: ENV['DB_HOST'],
+        password: ENV['DB_PASS'],
+        user: ENV['DB_USER'],
+        dbname: ENV['DB_NAME'],
+        port: ENV['DB_PORT']
+      )
+    end
+
+    def show
+      connection.exec('SELECT * FROM MEMO ORDER BY ID')
+    end
+
+    def find(id)
+      connection.exec('SELECT * FROM MEMO WHERE id = $1 ORDER BY ID', [id])[0]
+    end
+
+    def create(title, body)
+      connection.exec('INSERT INTO MEMO (title, body) VALUES($1, $2) ', [title, body])
+    end
+
+    def delete(id)
+      connection.exec('DELETE FROM MEMO WHERE id = $1 ', [id])
```

Comment:
```suggestion
      connection.exec('DELETE FROM MEMO WHERE id = $1', [id])
```

行末のスペースは不要です。(updateも同様)